envs/example/ci-allinone/group_vars/all.yml

```diff
@@ -0,0 +1,15 @@
+---
+stack_env: example_ci_allinone
+floating_ip: 172.16.0.100
+
+xtradb:
+  sst_auth_password: asdf
+
+percona:
+  replication: False
+
+ceilometer:
+  enabled: True
```

Comment:
Why is ceilometer being turned on for an allinone? That's going to put a lot of pressure on the VM, given what ceilometer brings in.